docs/user/ppl/functions/datetime.rst

```diff
@@ -872,6 +944,29 @@ Examples::
     +-----------------------------------+
 
 
+GET_FORMAT
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: Returns a string value containing string format specifiers based on the input arguments.
+
+Argument type: TYPE, STRING
+TYPE must be one of the following tokens: [DATE, TIME, DATETIME, TIMESTAMP].
```

Comment:
Same here missing new lines. You can view rich diff in the PR.

Reply:
Updated in https://github.com/Bit-Quill/opensearch-project-sql/pull/283/commits/872ece9c46db0eab82c2f34ed3cf7d973947c02a